include/dali/core/span.h

```diff
@@ -58,10 +59,30 @@ class span {
   >>
   DALI_HOST_DEV constexpr span(const span<U, Extent>& s) noexcept : data_(s.data()) {}
 
+
   DALI_HOST_DEV constexpr span(pointer firstElem, pointer lastElem) : data_(firstElem) {
     /* assert(lastElem - firstElem == Extent); */
   }
 
+  /**
+   * @remarks Container is not a specialization of span,
```

Comment:
This is like a doc for what Container is not instead of what this function is supposed to do.